### ISSUE TITLE
Correct alt text for Stanford Libraries logo link

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/exhibits#2471]
### Current behavior

The "Stanford Libraries" logo links to the [Exhibits homepage](https://exhibits.stanford.edu/), but has misleading alt text: "Spotlight Accessibility for SODA - Spotlight at Stanford".

### Steps to reproduce

Example page: [Spotlight Accessibility for SODA](https://exhibits.stanford.edu/a11y-for-soda). Inspect the logo link text.

### Possible solution

WCAG Level A: [Understanding Success Criterion 1.1.1: Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html).

Update the link text to "Stanford Libraries - Spotlight at Stanford".

**Note**: To be consistent with [other](https://archives.stanford.edu/) [applications](https://earthworks.stanford.edu/), the "Stanford Libraries" logo should link to the Stanford Libraries [homepage](https://library.stanford.edu/). Design and Product should discuss how to address this inconsistency in Exhibits. One possibility may be to specify "Spotlight" somewhere in the header logo design.

### Reported by SODA
> Violation: Ensure alternative text for image links is informative
> Severity: Major
>
> [Issue]
> The logo in the header, which is placed as a background image, has an incorrect text link alternative. The text is the same as the h1 of the page, but in this context that is incorrect as the link takes you to the homepage for this site. 
> 
> [User Impact]
>  Links require meaningful alt attributes which describe their target.
> 
> [Code Reference]
> `<a class="mb-0 navbar-brand navbar-logo" href="/">Spotlight Accessibility for SODA - Spotlight at Stanford</a>`
> 
> [Recommendation]
> Rewrite the text so that it indicates where the link is taking you, in this case the homepage for exhibits. 
> 
> [Compliant Code Example]
> `<a class="mb-0 navbar-brand navbar-logo" href="/">Stanford Libraries Exhibits</a>`
